lib/record_list.rb

```diff
@@ -15,6 +15,7 @@ def each(&block)
 
       response
         .fetch("records")
+        .select {|data| data.fetch("source").fetch("id") == EXTERNAL_CONTENT_SOURCE_ID }
         .select {|data| data.fetch("type").fetch("id") == EXTERNAL_CONTENT_TYPE_ID }
```

Comment:
I guess this approach will still slow down as more and more content is added to External Content – ideally we'd filter at the API level. But this should work :+1:

Reply:
Agreed. There's no option in the API at the moment to filter by source or type ID but that would be ideal.